### DISCUSSION TITLE
Options to change indent width and line height

### DISCUSSION THREAD

**Discussion by Jason-Abbott** [vincode-io/Zavala#191] [Ideas]
Great app. I’m very appreciative. I like the line height and indent size in iOS but would really like an option to make the indents about triple and the line heights half on MacOS. 

I’ve tried a lot of different font settings for the levels (great feature, BTW 👍 ) on MacOS but with no option to change the shallow indents and large line spacing, I still find it hard to know where I’m at, _at a glance_, compared to other outliners.

**Comment by vincode-io** [comment]
I plan on adding a few appearance customizations in the upcoming 2.1 release. One of those is line spacing #153. I'll try to get adjustable indenting in as well. It is being tracked as #192.
